docs/source/debugging.rst

```diff
@@ -179,7 +179,8 @@ launching Dask on your own, they will probably dump to the screen unless you
 <https://en.wikipedia.org/wiki/Redirection_(computing)#Redirecting_to_and_from_the_standard_file_handles>`_
 .
 
-You can control the logging verbosity in the ``~/.dask/config.yaml`` file.
+You can control the logging verbosity in the :doc:`configuration`, for example,
+the ``~/.dask/config.yaml`` file.
```

Comment:
I know this was here already, but `~/.dask/config.yaml` has been [deprecated](https://docs.dask.org/en/latest/configuration.html#yaml-files) and so now may be a good time to update the example config file given here

```suggestion
the ``~/.config/dask`` file.
```